qiskit_machine_learning/circuit/library/raw_feature_vector.py

```diff
@@ -65,15 +69,15 @@ class RawFeatureVector(BlueprintCircuit):
     def __init__(self, feature_dimension: Optional[int]) -> None:
         """
         Args:
-            feature_dimension: The feature dimension and number of qubits.
+            feature_dimension: The feature dimension from which the number of
+                                qubits is inferred as ``n_qubits = log2(feature_dim)``
 
         """
         super().__init__()
 
-        self._num_qubits = None
         self._ordered_parameters = ParameterVector("x")
-
-        if feature_dimension:
+        self.num_qubits = None
```

Comment:
Is this needed? If you read back, i.e. do a self.num_qubits does that not end up going to the property getter in QuantumCircuit? The _num_qubits and holding None I guess was done as a independent instance var on this class, same as in n_local. Hence the overridding of the num_qubits property - which now just calls super and as such could be deleted if thats what needed.

Now I note in the num_qubits setter that it does not update feature dimension (setting feature dim does update num qubits so it seems asymmetric in that regard). I am just noting that check_configuration does so against feature_dimension so hoepfully it never calls resize when that is still None.

Reply:
Agree, this is not required anymore, this is a legacy from `_num_qubits`.